owncloudApp/src/main/java/com/owncloud/android/utils/MdmConfigurations.kt

```diff
@@ -40,6 +41,7 @@ const val CONFIGURATION_OAUTH2_OPEN_ID_PROMPT = "oauth2_open_id_prompt"
     CONFIGURATION_ALLOW_SCREENSHOTS,
     CONFIGURATION_OAUTH2_OPEN_ID_SCOPE,
     CONFIGURATION_OAUTH2_OPEN_ID_PROMPT,
+    CONFIGURATION_DEVICE_PROTECTION,
```

Comment:
There's still some things to do with this:
- You need to add it to `managed_configurations.xml`
- You need to cache it in `SplashActivity.kt`

Take a look at the others configurations here to see how they are managed

Reply:
Added.